smelli/classes.py

```diff
@@ -155,6 +161,16 @@ def load_sm_covariances(self, folder):
             filename = os.path.join(folder, name + '.p')
             flh.sm_covariance.load(filename)
 
+    def save_exp_covariances(self, folder):
```

Comment:
Can you please add a doc string?

Reply:
Like for `save_sm_covariances`, this method is mentioned in the doc string of `GlobalLikelihood` (in line 42). Don't you think that's enough?